src/main/src/main/java/org/geoserver/ows/QuickTemplate.java

```diff
@@ -0,0 +1,47 @@
+/* (c) 2017 Open Source Geospatial Foundation - all rights reserved
```

Comment:
If this is a new file, it needs the correct date in the copyright header. @lmoran I think you have signed a contributor agreement; please confirm.

Reply:
It's a copy of one class I made for the OpenSearch for EO.

Reply:
If the class is a copy from elsewhere in the codebase, then it best to leave the copyright date and author tag the same.